hack/generator/pkg/astmodel/allof_type.go

```diff
@@ -159,3 +159,19 @@ func (allOf *AllOfType) String() string {
 
 	return fmt.Sprintf("(allOf: %s)", strings.Join(subStrings, ", "))
 }
+
+// DebugDescription adds a description of the current AnyOf type to the passed builder
+// builder receives the full description, including nested types
+// types is a dictionary for resolving named types
+func (allOf *AllOfType) WriteDebugDescription(builder *strings.Builder, types Types) {
+	builder.WriteString("AllOf[")
+	first := true
+	allOf.types.ForEach(func(t Type, ix int) {
+		if !first {
+			builder.WriteString("|")
+		}
+		t.WriteDebugDescription(builder, types)
```

Comment:
minor: Wondering if newlines would help any in displaying things?

Reply:
They frequently end up embedded within an `error` message, so I suspect newlines would cause more problems than they fix.